src/askem_beaker/contexts/mira_model_edit/context.py

```diff
@@ -589,6 +589,9 @@ async def stratify_request(self, message):
 		params_to_preserve = content.get("params_to_preserve")
 		cartesian_control = content.get("cartesian_control")
 		structure = content.get("structure")
+		add_param_factor = content.get("add_param_factor")
```

Comment:
Is there a corresponding client update?
We will want to change https://github.com/DARPA-ASKEM/terarium/blob/main/packages/client/hmi-client/src/components/workflow/ops/stratify-mira/tera-stratify-drilldown.vue#L213